### PR TITLE
aggregate aprs

### PR DESCRIPTION
**Pull request by oxSaturn**
Similar to https://github.com/Velocimeter/frontend/pull/258, unit test included.

---

### Focus of the PR:
This PR focuses on aggregating `aprRange` for the same token symbol in the `useGaugeApr` hook.

### Detailed summary:
- Added `aggregateApr` function to aggregate `aprRange` for the same token symbol.
- Modified `useGaugeApr` hook to use `aggregateApr` function.
- Added tests for `aggregateApr` function.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: components/options/stake.tsx [hunk 1]
```diff
@@ -225,14 +225,15 @@ export function Stake() {
...
                       ? `${formatCurrency(
                           token.aprRange[0]
-                        )} % - ${formatCurrency(token.aprRange[1])} %`
+                        )} - ${formatCurrency(token.aprRange[1])}%`
                       : `
-                  ${formatCurrency(token.apr)} %
+                  ${formatCurrency(token.apr)}%
                   `}
                   </div>
+                  {token.symbol}
```
Comment (oxSaturn) [inline components/options/stake.tsx:236]: Obviously people found it difficult to tell which is which:

CleanShot-KYkjOrjU@2x

File: components/options/stake.tsx [hunk 2]
```diff
@@ -225,14 +225,15 @@ export function Stake() {
...
                   <img
                     src={token.logoUrl ?? "/tokens/unknown-logo.png"}
                     title={token.symbol}
```
Comment (oxSaturn) [inline components/options/stake.tsx:239]: Really would love to get rid of the icons, especially when the size looks visually differently:

CleanShot-q69oUxlo@2x

Comment (t0rbik) [reply components/options/stake.tsx:239]: lets change it to symbols

Comment (oxSaturn) [reply components/options/stake.tsx:239]: `symbol` already included, see comment here https://github.com/Velocimeter/frontend/pull/269#discussion_r1263246295

Comment (oxSaturn) [reply components/options/stake.tsx:239]: Here's how it looks in this PR:

CleanShot-mbpwbP02@2x

Comment (t0rbik) [reply components/options/stake.tsx:239]: hm. yeah well... dunks is a fan of icons. we removed it in liquidity page maybe can leave it here haha

Comment (oxSaturn) [reply components/options/stake.tsx:239]: lol, ok.

Comment (t0rbik) [reply components/options/stake.tsx:239]: can revisit

**Pull request merged**